cargo-n64/Cargo.toml

```diff
@@ -21,6 +21,7 @@ goblin = { version = "0.2", default-features = false, features = ["std", "elf32"
 itertools = "0.8"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
+cargo-xbuild = "=0.5.25"
```

Comment:
I'm happy with a loose semver, here.

Even though a breaking change ended up in a patch release last time. But that's ok! It happens. That's what the CI cron job is for.

Also, this is really nitpicky, but would you mind maintaining alphabetical sorting in this list?

Reply:
I'm a stickler for alphabetical sorting too. 😁 Semver loosened and deps sorted.